test_scripts/Polices/build_options/031_ATF_P_TC_PTU_Retry_Sequence_Retry_Timeout_Computation_HTTP.lua

```diff
@@ -177,15 +197,18 @@ end
 
 function Test:RegisterNotification()
   self.mobileSession:ExpectNotification("OnSystemRequest", { requestType = "HTTP" })
-  :Do(function()
-      log("SDL->MOB: OnSystemRequest")
+  :Do(function(_, data)
+      print("SDL->MOB1: OnSystemRequest, requestType: " .. data.payload.requestType)
```

Comment:
@istoimenova, Please add logic that only HTTP requests is taking into accout
Currently "OnSystemRequest" requests with any type will be caught.